### DISCUSSION TITLE
Link to docker-related announcements no longer correct?

### DISCUSSION THREAD

**Discussion by fsimonazzi** [dotnet/dotnet-docker#3630] [General]
### Describe the Bug

The dockerhub page indicates: "Watch [dotnet/announcements](https://github.com/dotnet/announcements/labels/Docker) for Docker-related .NET announcements.". However, the last announcement there is from 2019. What would be the appropriate way monitor releases?

![Screen Shot 2022-04-11 at 14 41 10](https://user-images.githubusercontent.com/1406470/162798708-9c4ec8b8-d71c-40b6-8369-752949e85000.png)

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.

**Comment by mthalman** [comment]
We'll get the readme updated. Announcements will be posted to the [dotnet-docker](https://github.com/dotnet/dotnet-docker) repo.

General announcements will be tagged with [announcement](https://github.com/dotnet/dotnet-docker/issues?q=is%3Aopen+is%3Aissue+label%3Aannouncement).

Release issues are tagged as [area-release](https://github.com/dotnet/dotnet-docker/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-release).
